state/workingset.go

```diff
@@ -568,6 +610,31 @@ func (ws *workingSet) handleVote(blockHeight uint64, votes []*action.Vote) error
 		if err != nil {
 			return err
 		}
+
+		if !disableGasCharge {
```

Comment:
Can we unify the code between transfer and vote

Reply:
it's slightly different, in transfer, we compare (gas+amount) with balance. in vote, we compare gas with balance.